### ISSUE TITLE
Unhandled escape codes since 530de00

### ISSUE THREAD

**Issue by b0o** [alacritty/alacritty#4941]
See the following screenshot for the expected output (left: 7291702f) and bad output (right: 530de00):

![2021-04-04_13-29-33_100002:lg:2](https://user-images.githubusercontent.com/21299126/113520699-d4c89b00-9549-11eb-848f-b6a86d7a0e8a.png)

The issue first appeared in 530de00, but it is still present in the latest master (cbcc129). I've opened an issue with the Lipgloss project which includes more information: https://github.com/charmbracelet/lipgloss/issues/5.

### System

OS: Arch Linux
WM: Sway (Occurs under both Wayland and XWayland)

**Comment by muesli** [comment]
Some more insight on this one: what's happening here is that the program queries the terminal with Xterm control sequences (OSC) for its current background color. The terminal responds by writing the response to stdout, and the "escape codes" we see here is such a response (an encoded color information).

**Comment by chrisduerr** [comment]
I'd assume this is a timing/order issue, since the response does not immediately follow the request anymore. Making this a duplicate of https://github.com/alacritty/alacritty/issues/4872. The response itself seems correct, so please let me know if this is still a problem after #4872 is resolved.

**Comment by muesli** [comment]
Thank you @chrisduerr, subscribed to #4872 and will report back as soon as it's resolved!

**Comment by meowgorithm** [comment]
Looks like this one is fixed now: https://github.com/alacritty/alacritty/pull/4988. I'm no longer seeing the issue in charmbracelet/lipgloss#5 in Alacritty built on `master`.
